hack/local-up-cluster.sh

```diff
@@ -71,21 +71,35 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 function usage {
             echo "This script starts a local kube cluster. "
             echo "Example 1: hack/local-up-cluster.sh -o _output/dockerized/bin/linux/amd64/ (run from docker output)"
-            echo "Example 2: hack/local-up-cluster.sh (build a local copy of the source)"
+            echo "Example 2: hack/local-up-cluster.sh -O (auto-guess the bin path for your platform)"
+            echo "Example 3: hack/local-up-cluster.sh (build a local copy of the source)"
+}
+
+# This function guesses where the existing cached binary build is for the `-o`
+# flag
+function guess_built_binary_path {
+  local kubelet_path=$(kube::util::find-binary "kubelet")
+  if [[ -z "${kubelet_path}" ]]; then
+    return
+  fi
+  echo -n "$(dirname "${kubelet_path}")"
```

Comment:
Verify that both kubectl and hyperkube exist here?  It is easy to build one but not the other.